run_jobs.py

```diff
@@ -118,15 +119,26 @@ def run_jobs(self) -> None:
 
 
 if __name__ == '__main__':
+    db_creator_obj = DBCreator(ENV['INVENTORY_DB'], ENV['APPEND_TABLE_NAMES'])
     how_started = os.environ.get('HOW_STARTED', None)
 
     if how_started == 'DOCKER_COMPOSE':
-        logger.info('Waiting for the MySQL turtle, hehe')
         # Wait for MySQL container to finish setting up
-        time.sleep(30.0)
+        logger.info('Waiting for the MySQL turtle')
+        waiting = True
+        while waiting:
+            time.sleep(3.0)
+            try:
+                db_creator_obj.set_up()
+                db_creator_obj.tear_down()
```

Comment:
When changing the loop condition to test `db_creator_obj.conn`, also move this call to `db_creator_obj.tear_down()` from inside the loop to just outside the loop.  That's what will leave the `.conn` property as non-`None`.

Reply:
Since the loop has changed from `while` to `for`, there's no need to move `db_creator_obj.tear_down()` out of the loop.